config/currency_provider_tracker.go

```diff
@@ -20,6 +20,168 @@ const (
 	trackingPeriod              = time.Hour * 24
 )
 
+// SupportedForexCurrencies defines a lookup table for all the supported
```

Comment:
- Should this be a const? Do these structs get filled with anything? The only reference I see to it is a check for the existence of a key in the map.
- I could see this in a separate file for readability.

Reply:
I wrote it based on `SupportedQuotes` and `SupportedForexCurrencies`, I think the empty struct is a place holder to be able to use a map as a lookup table for constant time key access. Also since they are maps they can't be set as consts. I agree with the separate file idea and also moved  `SupportedQuotes` and `SupportedForexCurrencies` into it; see `supported_assets.go`.